### DISCUSSION TITLE
Security issue with GitHub: A former team member who was removed from an organization can still push commits to a private repo

### DISCUSSION THREAD

**Discussion by alon289** [community/community#83981] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Th user left the company a couple of weeks ago and was immediately removed from the repository, from the team and from the organization. Despite that, a computer that was logged in through a previous Token was able to push commits to the private repository.

**Comment by AnasteishaSk** [comment]
you can check complete user access to your repos - accesshub.app
